plugins/exporter.koplugin/target/my_clippings.lua

```diff
@@ -11,24 +11,35 @@ local ClippingsExporter = require("base"):new {
 }
 
 local function format(booknotes)
-    local content = ""
+    local tbl = {}
+
     for ___, entry in ipairs(booknotes) do
         for ____, clipping in ipairs(entry) do
             if booknotes.title and clipping.text then
-                content = content .. booknotes.title .. "\n"
-                local header = T(_("- Your highlight on page %1 | Added on %2"), clipping.page, os.date("%A, %B %d, %Y %I:%M:%S %p", clipping.time)) .. "\n\n"
-                content = content .. header
-                content = content .. clipping.text
-                content = content .. "\n==========\n"
+                table.insert(tbl, booknotes.title .. "\n")
+
+                local header = T(_("- Your highlight on page %1 | Added on %2"), clipping.page,
+                    os.date("%A, %B %d, %Y %I:%M:%S %p", clipping.time)) .. "\n\n"
+                table.insert(tbl, header)
+
+                table.insert(tbl, clipping.text)
+                table.insert(tbl, "\n==========\n")
+
                 if clipping.note then
-                    content = content .. booknotes.title .. "\n"
-                    header = T(_("- Your note on page %1 | Added on %2"), clipping.page, os.date("%A, %B %d, %Y %I:%M:%S %p", clipping.time)) .. "\n\n"
-                    content = content .. header .. clipping.note
-                    content = content .. "\n==========\n"
+                    table.insert(tbl, booknotes.title .. "\n")
+
+                    header = T(_("- Your note on page %1 | Added on %2"), clipping.page,
+                        os.date("%A, %B %d, %Y %I:%M:%S %p", clipping.time)) .. "\n\n"
+                    table.insert(tbl, header)
+
+                    table.insert(tbl, clipping.note)
+                    table.insert(tbl, "\n==========\n")
                 end
             end
         end
     end
+
+    local content = table.concat(tbl)
     return content
```

Comment:
Reads a bit funny. ;-)

Reply:
Oh gosh ahah thanks for pointing it out!